tests/test_resource.py

```diff
@@ -0,0 +1,12 @@
+from tutor.models.course import Resource
+
+
+def test_add_new_resource():
```

Comment:
As @ifireball mention in the last session - what are you trying to test? it seems that you are testing the ORM.
This is a unit testing- you don't need to test ORM, it's not yours and someone else already tested it 

remove that test and add a real UT

Reply:
I tried to test the addition of a new resource. Is this what you meant in "testing the ORM"?

Reply:
There's already a merged test_source.py at which I think contains the code on this PR(by looking at the conflict). IMHO I don't see a reason to continue this one. please correct me if I'm wrong.